docs/preview_conversion_analyzer.md

```diff
@@ -0,0 +1,129 @@
+<!-- The purpose of this spec is to describe a new feature and
+its APIs that make up a new feature in WinUI. -->
+
+<!-- There are two audiences for the spec. The first are people
+that want to evaluate and give feedback on the API, as part of
+the submission process.  When it's complete
+it will be incorporated into the public documentation at
+docs.microsoft.com (http://docs.microsoft.com/uwp/toolkits/winui/).
+Hopefully we'll be able to copy it mostly verbatim.
+So the second audience is everyone that reads there to learn how
+and why to use this API. -->
+# Converting To WinUI3 Using CSharp Conversion Analyzers
+
+## Background
+<!-- Use this section to provide background context for the new API(s) 
+in this spec. -->
+The purpose of this project is to create a porting solution in the form of Code Analyzers/Fixes to allow Developers to convert WinUI2 projects to the new WinUI3 format. As Microsoft moves toward a purely public SDK, its Xaml UI components are being lifted out of the OS and deployed in WinUI3.
```

Comment:
The "As Microsoft moves towards a purely public SDK" isn't relevant to this tool (that's more about the process of moving to open source). I think what's relevant to the tool is that WinUI2 ships as part of the OS, WinUI3 ships out of the OS as a NuGet package.

Reply:
Updated background